README.md

```diff
@@ -68,14 +68,14 @@ Notes:
 
 - For devfile hosts (environment variable or config file) specify a remote url or a local folder structure and `index.json` that follows the `devfile/api` spec.
 - You can have infinitely many sources as long as the name for the source repository is different.
-- There are two types of source types, url and stacks.
+- There are two types of source types, url and local.
   - url is for specifying a remote hosts.
-  - stacks is for specifying a local hosts. NOTE: Stacks MUST have `index.json` under the root path specified, an example is below.
+  - local is for specifying a local hosts. NOTE: Local MUST have `index.json` under the root path specified, an example is below.
 
-`/stacks`
+`/devfiles`
 
 ```
-+-- stacks
++-- devfiles
```

Comment:
idk, it was kind of nice to be stacks, since thats what the dir was called in devfile/registry and you could copy-paste it https://github.com/devfile/registry/tree/main/stacks

Reply:
The folder name can be any name. I just needed to rename the config var to something besides `stacks` or people would just assume it would only support local stacks and not local samples.